src/middlewared/middlewared/service.py

```diff
@@ -69,37 +70,88 @@ class throttle(object):
     on some key (possibly argument of the method).
     """
 
-    def __init__(self, seconds=0, condition=None, exc_class=RuntimeError):
+    def __init__(self, seconds=0, condition=None, exc_class=RuntimeError, max_waiters=10):
```

Comment:
These changes @themylogin, are they related ?

Reply:
Oh, God, thank you, I have no idea how did it get there